config/locales/translations/zendesk_apps_support.yml

```diff
@@ -101,6 +101,10 @@ parts:
       key: "txt.apps.admin.error.app_build.excessive_requirements"
       title: "App builder job: requirements file contains too many requirements"
       value: "The requirements.json file contains too many requirements. The current limit is %{max} requirements. This app has %{count} requirements."
+  - translation:
+      key: "txt.apps.admin.error.app_build.excessive_custom_objects_requirements"
+      title: "App builder job: requirements file contains too many custom objects requirements"
+      value: "The requirements.json file contains too many custom objects requirements. The current limit is %{max} requirements. This app has %{count} requirements."
```

Comment:
They will ask for a screenshot here for new strings

Reply:
I've added one, thanks